tests/Serilog.Sinks.RabbitMQ.Tests.Integration/RabbitMqClientTest.cs

```diff
@@ -0,0 +1,93 @@
+namespace Serilog.Sinks.RabbitMQ.Tests.Integration
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using global::RabbitMQ.Client;
+    using global::RabbitMQ.Client.Events;
+    using global::RabbitMQ.Client.Exceptions;
+    using Serilog.Sinks.RabbitMQ.Sinks.RabbitMQ;
+    using Xunit;
+
+    /// <summary>
+    ///   Tests for <see cref="RabbitMQClient" />.
+    /// </summary>
+    [Collection("Sequential")]
+    public sealed class RabbitMqClientTest : IDisposable
+    {
+        private const string QueueName = "serilog-sink-queue";
+        private const string HostName = "rabbitmq";
+
+        private readonly RabbitMQClient client = new RabbitMQClient(new RabbitMQClientConfiguration
+        {
+            Port = 5672,
+            DeliveryMode = RabbitMQDeliveryMode.Durable,
+            Exchange = "serilog-sink-exchange",
+            Username = "guest",
+            Password = "guest",
+            ExchangeType = "fanout",
+            Hostnames = { HostName },
+        });
+
+        private IConnection connection;
+        private IModel channel;
+
+        /// <summary>
+        ///   Consumer should receive a message after calling Publish.
+        /// </summary>
+        /// <returns>A task that represents the asynchronous operation.</returns>.
+        [Fact]
+        public async Task Publish_SingleMessage_ConsumerReceivesMessage()
```

Comment:
This method name is a bit unclear to me and I am actually a little bit confused about what this method is testing. 
I personally prefer this test naming convention "Should_ExpectedBehavior_When_StateUnderTest". In this situation would be something like "Consumer_ShouldReceiveSingleMessage_WhenMessageSuccessfullyPublished". I guess that it would be great if we can agree on a standard naming convention. What do you think @sonicjolt

Reply:
@tsvetelintsonev I'm not religious on this point, but I think it's good to follow general patterns like Arrange, Act, Assert.. And I agree that it's a good idea that we agree on a standard. For this PR, I suggest that @huan086 perhaps considers if the naming can be improved, and perhaps section off the test following arrange, act, assert, .. just for the sake structure.

Let's make an issue to discuss a naming convention. @huan086 Do you have any insight that you would like to share ?